pkg/provider/gke/gke.go

```diff
@@ -577,3 +581,12 @@ func (c *GKE) ResourceDelete(*kingpin.ParseContext) error {
 	}
 	return nil
 }
+
+// GetDefaultDeploymentVars shows default deployment var
+func (c *GKE) GetDefaultDeploymentVars(parseContext *kingpin.ParseContext) error {
+	fmt.Print("Key : Value \n")
```

Comment:
We can just call this function info, that way we would be able to add more provider specific info later if we want. In place of `Key: Value`, just writing `DeploymentVars` would do imo. but feel free to keep it this way if you feel strongly about this :)